e2e/framework/manager/rook_test_infra.go

```diff
@@ -0,0 +1,407 @@
+package rook_test_infra
```

Comment:
i would just call this infra.go. You already know you're in a rook test subfolder.